src/test/java/com/google/cloud/spanner/pgadapter/python/ITPsycopg2Test.java

```diff
@@ -20,16 +20,20 @@
 import com.google.cloud.spanner.pgadapter.IntegrationTest;
 import com.google.cloud.spanner.pgadapter.PgAdapterTestEnv;
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Collections;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.junit.runners.MethodSorters;
 
 @RunWith(JUnit4.class)
 @Category(IntegrationTest.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
I know that we use this some places in the Cloud Spanner JDBC driver, but generally speaking, we should try to avoid this in new code. This makes each test case potentially dependent on all the previous test cases, which can make it very difficult to debug when something goes wrong. It also makes it more difficult to add new test cases or modify existing ones.

It is therefore better to insert/update the data that is needed for a specific test as part of the setup of that specific test, instead of relying on data from a previous test.

Note: Re-using the same table for all tests in a test class is OK, as creating a table can take a little while on Cloud Spanner.

Reply:
Okay. I've changed the code in such a way that the order of tests will not matter now.

Reply:
Thanks, LGTM